saleor/account/models.py

```diff
@@ -282,3 +282,27 @@ class Meta:
 
     def __repr__(self):
         return f"{self.__class__.__name__}(type={self.type!r}, user={self.user!r})"
+
+
+class StaffNotification(models.Model):
```

Comment:
I wonder if the name won't be missleading? It sounds like it is containing the notifications, and no the user preferences.

Reply:
Maybe `StaffNotificationRecipient` since this model is about storing the recipient of emails - either a user instance or just an email address?

Reply:
It could also be just `NotificationRecipient`. Not sure if it matters for the model name that it's about staff users. Then we could rename `staff_email` field to just `email`. In the API it would be restricted to staff, but maybe models don't have to know about that.

Reply:
cc @NyanKiyoshi

Reply:
Sounds perfect and way better!

Reply:
decided that first propose (`StaffNotificationRecipient `) is good enough

Reply:
Isn't `NotificationRecipient` better?